lib/widgets/rules_buttons.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+
+import 'game_view.dart';
+
+class RulesChangingButtons extends StatefulWidget {
+  const RulesChangingButtons({super.key});
+
+  @override
+  RulesChangingButtonsState createState() => RulesChangingButtonsState();
+}
+
+class RulesChangingButtonsState extends State<RulesChangingButtons> {
+  //int rules = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        const Text('Get'),
+        const SizedBox(width: 20),
+        SizedBox(
+          width: 87,
+          height: 30, // Replace with the desired width
+          child: OutlinedButton(
+            onPressed: () {},
+            style: ButtonStyle(
+              side: MaterialStateProperty.all(
+                const BorderSide(color: Colors.black), // Set the outline color
+              ),
+              shape: MaterialStateProperty.all(
+                RoundedRectangleBorder(
+                  borderRadius:
+                      BorderRadius.circular(5.0), // Set the border radius
+                ),
+              ),
+            ),
+            child: DropdownButton<int>(
+              value: inRowValue,
+              items: [
+                const DropdownMenuItem<int>(
+                  value: 3,
+                  child: Text('3'),
+                ),
+                if (boardSize > 3)
+                  const DropdownMenuItem<int>(
+                    value: 4,
+                    child: Text('4'),
+                  ),
+                if (boardSize > 4)
+                  const DropdownMenuItem<int>(
+                    value: 5,
```

Comment:
Please, make values to be 2 to 10 and add them in loop.

Reply:
Now it's made other way